src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -227,6 +71,15 @@ function PolicyAccountingPage({policy}: PolicyAccountingPageProps) {
 
     const accountingIntegrations = Object.values(CONST.POLICY.CONNECTIONS.NAME);
     const connectedIntegration = getConnectedIntegration(policy, accountingIntegrations) ?? connectionSyncProgress?.connectionName;
+    const synchronizationError = connectedIntegration && hasSynchronizationError(policy, connectedIntegration, isSyncInProgress);
+
+    // Enter credentials item shouldn't be shown for SageIntacct and NetSuite integrations
+    const shouldShowEnterCredentials =
+        connectedIntegration &&
+        !!synchronizationError &&
+        isAuthenticationError(policy, connectedIntegration) &&
```

Comment:
`isAuthenticationError` return false even we didn't sync successfully is the RCA of the issue I've just posted

<img width="1331" alt="Screenshot 2024-08-12 at 17 30 57" src="https://github.com/user-attachments/assets/4caf92cf-2b08-4766-a7d3-c50701a726a6">

Reply:
> `isAuthenticationError` return false even we didn't sync successfully is the RCA of [the issue I've just posted](https://github.com/Expensify/App/pull/45251#issuecomment-2283579930)

@hungvu193 That's right 👍 I already [asked](https://github.com/Expensify/App/pull/45251#issuecomment-2275925735) @arosiclair about this issue and now we're waiting for some [backend changes](https://github.com/Expensify/App/pull/45251#issuecomment-2276971381)

Reply:
In the meantime, we can set `isAuthenticationError` manually (to check the rest of the flow) 😅